src/udfs/video_action_classification.py

```diff
@@ -0,0 +1,210 @@
+from src.models.catalog.frame_info import FrameInfo
+from src.models.catalog.properties import VideoFormat, ColorSpace
+from src.models.catalog.video_info import VideoMetaInfo
+from src.models.storage.frame      import Frame
+from src.models.storage.batch      import FrameBatch
+from src.models.inference.classifier_prediction import Prediction
+
+from src.loaders.video_loader import SimpleVideoLoader
+from src.udfs.abstract_udfs import AbstractClassifierUDF
+
+
+from tensorflow.python.keras.models import Sequential
+from tensorflow.python.keras.layers import Dense, Conv2D, Flatten
+
+import cv2
+
+from typing import List, Tuple
+from glob import glob
+import numpy as np
+import random
+import os
+
+
+class ActionClassificationLoader:
+    def __init__(self, path):

```

Comment:
Ideally, the loaders shouldn't be in UDFs. We can remove this from here and maybe use AbstractVideoLoader already defined in storage.